megamek/src/megamek/client/ui/swing/MiniReportDisplay.java

```diff
@@ -252,12 +253,13 @@ private void updateQuickChoice() {
         for (String keyword : keywords) {
             comboQuick.addItem(keyword);
         }
-        if (comboQuick.getItemCount() == 1) {
-            comboQuick.setEnabled(false);
-        }
         comboQuick.setSelectedItem(lastChoice);
-        if (comboQuick.getSelectedIndex() < 0) {
-            comboQuick.setSelectedIndex(0);
+        if (comboQuick.getItemCount() <= 1) {
+            comboQuick.setEnabled(false);
+        } else {
+            if (comboQuick.getSelectedIndex() < 0) {
```

Comment:
Same here

Reply:
think that is for the case when your lastChoice was deleted from the list, to just select the first element